objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -250,6 +259,8 @@ boolean createNeuralNet(Map<String, ?> neuralNetConfig ) {
                 neuralNetType = YOLO_PROCESSOR_FQCN;
             } else if (neuralNetType.equals(NONE)) {
                 neuralNetType = NO_PROCESSOR_FQCN;
+            } else if (neuralNetType.equals(TEST)) {
+                neuralNetType = TEST_PROCESSOR_FQCN;
             } else if (neuralNetType.equals(DEFAULT)) {
                 neuralNetType = DEFAULT_NEURAL_NET;
```

Comment:
I take it that users cannot add their own here -- they have to add a NeuralNetInterface?  Though we've never had anyone ask (& it seems unlikely in the near term), I thought we advertised the ability.  Maybe they have to go change this code.  But it seems like something we should be able to handle in configuration & dynamic loading.

Add an issue...

Reply:
As it stands, the users would have to add their own NeuralNetInterface implementation, and then edit this portion of the code in order to use it.

I will add an issue to make the second part easier, since they shouldn't have to edit this code in order to actually use their implementation.